articles/constants-and-variables.html

```diff
@@ -922,8 +931,19 @@ <h3>Explicit Conversions On Non-Constant Numeric Values</h3>
 (towards zero) when it is converted to the integer type.
 </p>
 
-In the following example, the intended implicit conversions
-at line <i>4</i> and line <i>11</i> both don't work.
+<p>
+In Go, two typed values of two different basic types
+can't be assigned to each other.
+In other words, the types of the destination and source values in an
+assignment must be identical if the two values are both basic values.
+If the type of the source basic value is not same as the type of
+the destination basic value, then the source value must be explicitly
+converted to the type of the destination value.
+</p>
+
+<p>In the following example, the intended implicit conversions
+at line <i>4</i> and line <i>11</i> both don't work.</p>
+

```

Comment:
I noticed you move some texts from above to here. I think it is good and more relevant.
However, the texts above are not removed?

Reply:
I'm sorry, I don't understand !

Reply:
I mean you forgot delete the first appearance of the following texts"
```
<p>
In Go, two typed values of two different basic types
can't be assigned to each other.
In other words, the types of the destination and source values in an
assignment must be identical if the two values are both basic values.
If the type of the source basic value is not same as the type of
the destination basic value, then the source value must be explicitly
converted to the type of the destination value.
</p>
```

Reply:
Aha, maybe it was move here to above sometime by me, :)